docs/harvesting.md

```diff
@@ -89,6 +89,12 @@ optional arguments:
 ## Backends
 
 `udata` comes with 3 harvest backends but you can implement your own backend.
```

Comment:
Would be nice to list the existing backends in the README here, no?

Reply:
yes indeed, i'll add the ones i'm aware of ;)